components/EmailValidator.vue

```diff
@@ -141,6 +156,11 @@ export default {
         return 'Please enter a valid email address.'
       }
 
+      if (this.validateDomain) {
+        const result = await this.checkValidDomain(value);
+        return result || 'Invalid email domain';
```

Comment:
Please change `Invalid email domain` to `Please check your email domain - maybe you've made a typo?`.